lib/components/molecules/Popover/Popover.story.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+// import { action } from '@storybook/addon-actions';
+import {
+  defaultVisiblePopover,
+  hiddenPopover,
+  ListPopover,
+  noCloseBtnPopover,
+} from './Popover.mock';
+
+// Import Styled Component to showcase variations
+import Popover, { PopoverVanilla } from '.';
+
+storiesOf('Molecules', module).addWithChapters('Popover', {
+  chapters: [
+    {
+      title: `Helloooooo`,
+      sections: [
+        {
+          sectionFn: () => (
+            <PopoverVanilla
+              {...hiddenPopover}
+              className={`hide-default-sample ${hiddenPopover.className}`}
+            />
+          ),
+          options: {
+            showSource: true,
+            allowSourceToggling: true,
+            showPropTables: true,
+            allowPropTablesToggling: true,
+          },
+        },
+      ],
+    },
+    {
+      title: 'Popover Variations',
+      sections: [
+        {
+          title: '[isVisible=true] variation',
+          sectionFn: () => (
+            <Popover {...defaultVisiblePopover}>
+              <button>Click to Open</button>
+            </Popover>
+          ),
+        },
+        {
+          title: '[isVisible=false] variation',
+          sectionFn: () => (
+            <Popover {...hiddenPopover}>
+              <button>Click to Open</button>
+            </Popover>
+          ),
+        },
+        {
+          title: 'List(Component) variation',
+          sectionFn: () => (
+            <Popover {...ListPopover}>
+              <button>Click to Open</button>
+            </Popover>
+          ),
+        },
+        {
+          title: 'Optional close button variation',
+          sectionFn: () => (
+            <Popover {...noCloseBtnPopover}>
+              <button>Click to Open</button>
```

Comment:
See this is not really right, the content which is less is placed in the body of component (being picked as children) and the content which really is body gets set as prop. This is a wrong design pattern and would always confuse react developers. Can you please change it, i had given this comment earlier too. 

So use the component body to set the Popover content and defined prop for supplying heading and trigger element/content.